### PR TITLE
Fix the correct upstream.url

### PR DESCRIPTION
**Pull request by ptahchiev**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>jqm-autocomplete</artifactId>
-    <version>1.5.2-SNAPSHOT</version>
+    <version>1.5.3-SNAPSHOT</version>
```
Comment (jamesward) [inline pom.xml]: This should be `1.5.2-1-SNAPSHOT` for this patch update.

Comment (ptahchiev) [inline pom.xml]: I changed it

**Comment by jamesward** [comment]
This won't auto-merge.  Can you rebase on master?  Then force push to your remote and it will update this PR.

**Comment by ptahchiev** [comment]
I think I am on master already.. I don't need to rebase

**Comment by jamesward** [comment]
You need to rebase on my master.  Like:

```
git remote add upstream https://github.com/webjars/jqm-autocomplete.git
git fetch upstream
git rebase upstream/master
git push -f
```

**Comment by ptahchiev** [comment]
Thanks, I don't know git that well :)

**Comment by jamesward** [comment]
Looks like you got it!

**Pull request merged**